### DISCUSSION TITLE
Sort name field

### DISCUSSION THREAD

**Discussion by unchartedpopsicle** [mateusz-bak/openreads#724] [Ideas]
A manually entered author sort name would allow the more common last name sorting as discussed in #125 without odd formatting, and would prevent issues with multiple authors (#253).
To reduce editing time it could be pre-filled with the author info by default.

**Comment by jwildman16** [comment]
Also useful for travel guides (e.g., I would use "Lonely Planet" as the author for that guidebook series).
